Dockerfile

```diff
@@ -0,0 +1,20 @@
+# This is a multi-stage dockerfile for the zapp-backend
+
+# Build stage
+FROM node:14.4.0-alpine AS builder
+LABEL maintainer="nicklas@mediathekview.de"
+WORKDIR /usr/src/zapp-backend
+# copy app files
+COPY . .
+# install node packages
+RUN npm install
+
+# Run stage
+FROM node:14.4.0-alpine
+LABEL maintainer="nicklas@mediathekview.de"
+ENV PORT=3000
+EXPOSE $PORT
```

Comment:
In diesem speziellen Fall macht es keinen Sinn den Port zuerst in eine Environment Variable zu packen und dann zu exposen. Außer die Applikation greift auf die Umgebungsvariable zu. (Hab ich jetzt nicht geprüft)

Reply:
Ja, die Applikation nutzt den port. In der Readme unter Getting started heißt es:
```
4. Set PORT as an environment variable inside your console. Eg. PORT=3000 on linux or SET PORT=3000 on windows.
```